src/content/en/tools/workbox/modules/workbox-recipes.md

```diff
@@ -22,23 +22,36 @@ The offline fallback recipe allows your service worker to serve a web page, imag
 
 This recipe, by default, assumes the fallback page is `offline.html` and that there isn't an image or font fallback. See the [offline fallback options](/web/tools/workbox/reference-docs/latest/module-workbox-recipes#~offlineFallback) for a list of all configuration options.
 
+The offline fallback will only be applied if there's a matching [route](/web/tools/workbox/modules/workbox-routing) for a given request. When using offline fallback alongside other recipes, like for the page and image caches, a route will automatically be set up for those requests. When using offline fallback on its own, you will need to explicitly create a route that matches page and image requests. The simplest approach is to use the [`setDefaultHandler()`](/web/tools/workbox/modules/workbox-routing#set_a_default_handler) method to create a route that applies the [`NetworkOnly`](/web/tools/workbox/modules/workbox-strategies#network_only) strategy to all requests, as shown below.
```

Comment:
Sorry, can't do inline comments for whatever reason here. Just some wordsmithing:

`Other recipes, like the [page cache](#page_cache), [static resource cache](#static_resources_cache), or [image cache](#image_cache), set up routes for their respective caches, allowing the offline fallback to work as expected. If you're using the offline fallback recipe on its own, you'll need to create routes yourself. The simplest way to do is to use…`

Reply:
Is this something we want to include as an option in the offline fallback recipe itself? Is there a way to test for this to see if routes have been created and do it automatically instead of through a config option, if we were to include it?

Reply:
Changed the wording.

I can imagine enabling a default `NetworkOnly` handler being a useful option in the offline fallback recipe, but I am not sure about enabling that by default—definitely not in v6 at least, as it would be a breaking change.